apidump/apidump.go

```diff
@@ -226,6 +229,67 @@ func RotateLearnSession(args *Args, done <-chan struct{}, collectors []trace.Lea
 	}
 }
 
+// Goroutine to send telemetry, stop when "done" is closed.
+//
+// Prints a summary after a short delay.  This ensures that statistics will
+// appear in customer logs close to when the process is started.
+// Omits if args.StatsLogDelay is <= 0.
+//
+// Sends telemetry to the server on a regular basis.
+// Omits if args.TelemetryInterval is <= 0
+func SendTelemetry(args *Args, learnClient rest.LearnClient, backendSvc akid.ServiceID, dumpSummary *Summary, done <-chan struct{}) {
+	if args.StatsLogDelay <= 0 && args.TelemetryInterval <= 0 {
+		return
+	}
+
+	req := kgxapi.PostClientPacketCaptureStatsRequest{
+		ClientID:                  args.ClientID,
+		ObservedStartingAt:        time.Now().UTC(),
+		ObservedDurationInSeconds: args.StatsLogDelay,
+	}
+
+	// Upload "req" to the server.
+	send := func() {
+		req.PacketCountSummary = dumpSummary.FilterSummary.Summary(10)
+		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
Suggest using symbolic constants

Reply:
Fixed in https://github.com/akitasoftware/akita-cli/pull/137/commits/0e73da19024c7bcd9cc2c274e302e4f46b91fd00